cves/kernel/CVE-2012-2745.yml

```
@@ -55,7 +50,8 @@ description_instructions: |
 
   Your target audience is people just like you before you took any course in
   security
-description:
+description: There was a problem in a part of the code that handles creating new processes (like when you run a program). When a bad program asks to create a new process, this code gives the new process some wrong information. If a person with malicious intent uses a specially crafted program to make new processes, it could lead to overflow/denial of service.
```

A couple of points of feedback:
1. Mixed use of tenses in the first sentence.
```suggestion
description: There was a problem in a part of the code that handled creating new processes (like when you run a program). When a bad program asks to create a new process, this code gives the new process some wrong information. If a person with malicious intent uses a specially crafted program to make new processes, it could lead to overflow/denial of service.
```
2. Could you clarify what is meant by "this code gives the new process some wrong information"? What kind of information is it receiving that it's not supposed to or vice versa?

---

Thank you for fixing the tenses